src/octotree.js

```diff
@@ -11,8 +11,10 @@ $(document).ready(() => {
   }
 
   function createAdapter() {
-    const githubUrls = store.get(STORE.GHEURLS).split(/\n/).concat('github.com')
-    return ~githubUrls.indexOf(location.host)
+    const githubUrls = store.get(STORE.GHEURLS).split(/\n/)
+      .map((url) => url.replace(/(.*?:\/\/[^/]+)(.*)/, '$1'))
```

Comment:
I'm not sure if this is necessary. Before storing, the url is already [normalized](https://github.com/buunguyen/octotree/blob/master/src%2Fconfig%2Fchrome%2Fbackground.js#L44).